client/store/modules/activities/actions.js

```diff
@@ -0,0 +1,49 @@
+import {
+  fetch,
+  get,
+  reset,
+  save,
+  setBaseUrl,
+  update
+} from '../../helpers/actions';
```

Comment:
Maybe convert this multiple import into one line import to be more consistent because exports are one liner. And, also you can use alias `@/` from `poi.confing.js` and it should look like this: ` ... from '@/store/helpers/actions; `. Maybe he can create specific alias for `store` in poi ? @vladimyr